pyupgrade.py

```diff
@@ -1535,7 +1536,15 @@ def visit_Call(self, node: ast.Call) -> None:
         elif self._is_six(node.func, ('b', 'ensure_binary')):
             self.six_b.add(_ast_to_offset(node))
         elif self._is_six(node.func, SIX_CALLS) and not _starargs(node):
-            self.six_calls[_ast_to_offset(node)] = node
+            offset = _ast_to_offset(node)
+
+            if node.args and isinstance(
+                node.args[0],
+                (ast.BoolOp, ast.IfExp, ast.BinOp, ast.GeneratorExp),
+            ):
```

Comment:
I spent time to write the code for you in a comment with my preferred style, why did you pick a different style :S

Reply:
Well, I really thought a lot of how to answer this statement (which already may point into the direction of the problem with it), but I will throw them all away and just answer it with some technical reason: `black` overruled you.